### DISCUSSION TITLE
give me a idea!

### DISCUSSION THREAD

**Discussion by sunghyun61** [strongswan/strongswan#1213] [Q&A]
Hello.

I've setup IPSec conneciton StrongSwan with FortiGate.

My system can ping each site endpoint.

But, it can't connect any prot as TCP or UDP.
Endpoint has disabled firewall.

I've found one thing.
The firewall disabled at linux system(StrongSwan side), then all communications was reached successfully to endpoint each.

It need more setting at firewall on linux?
Please give me an idea.

**Comment by Thermi** [comment]
is it supposed to act as a router or an endpoint?
Maybe check the docs here https://docs.strongswan.org/docs/5.9/howtos/forwarding.html
